cmd/api/v1/audit.go

```diff
@@ -0,0 +1,180 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"connectrpc.com/connect"
+	apiv1 "github.com/metal-stack-cloud/api/go/api/v1"
+	"github.com/metal-stack-cloud/cli/cmd/config"
+	"github.com/metal-stack-cloud/cli/cmd/sorters"
+	"github.com/metal-stack/metal-lib/pkg/genericcli"
+	"github.com/metal-stack/metal-lib/pkg/genericcli/printers"
+	"github.com/metal-stack/metal-lib/pkg/pointer"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type audit struct {
+	c *config.Config
+}
+
+func newAuditCmd(c *config.Config) *cobra.Command {
+	a := &audit{
+		c: c,
+	}
+
+	cmdsConfig := &genericcli.CmdsConfig[*apiv1.AuditServiceGetRequest, *apiv1.AuditServiceGetRequest, *apiv1.AuditTrace]{
+		BinaryName:      config.BinaryName,
+		GenericCLI:      genericcli.NewGenericCLI[*apiv1.AuditServiceGetRequest, *apiv1.AuditServiceGetRequest, *apiv1.AuditTrace](a).WithFS(c.Fs),
+		Singular:        "audit trace",
+		Plural:          "audit traces",
+		Description:     "show audit traces of the api-server",
+		Sorter:          sorters.AuditSorter(),
+		OnlyCmds:        genericcli.OnlyCmds(genericcli.ListCmd, genericcli.DescribeCmd),
+		DescribePrinter: func() printers.Printer { return c.DescribePrinter },
+		ListPrinter:     func() printers.Printer { return c.ListPrinter },
+		ListCmdMutateFn: func(cmd *cobra.Command) {
+
+			cmd.Flags().String("request-id", "", "request id of the audit trace.")
+
+			cmd.Flags().String("from", "1h", "start of range of the audit traces. e.g. 1h, 10m, 2006-01-02 15:04:05")
```

Comment:
Let the server control defaults.

```suggestion
			cmd.Flags().String("from", "", "start of range of the audit traces. e.g. 1h, 10m, 2006-01-02 15:04:05")
```